chapters/07-build-management.Rmd

```diff
@@ -6,10 +6,31 @@
 - Pre-push running test, build, and check
 - Continuous integration using GitHub Actions
 
-
+<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
 ## Final exercise
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+The purpose of this exercise is for the participant to go through a cycle of iteration of:
+
+- 1) checking if their package succeeds with R CMD
+- 2) adding a function and the related steps
+- 3) check if the package succeeds R CMD with the new additions
+- 4) deploy a GitHub Action that performs the R CMD check every time they push to GitHub. 
 
+The individual steps look like this:
 
+1. Check that the package builds with the current functions
```

Comment:
```suggestion
1. Check the package in its current form.  Fix any notes, issues or warnings.
```
It might be nice if by this point their package actually fails `check()` in some predictable way.  So, they can make a fix (perhaps one they see in the chapter body) and verify `check()` succeeds.   Hard to predict exactly what this might be at this point...